test/sqllogictest/record.slt

```diff
@@ -59,3 +59,26 @@ query T
 SELECT abc FROM (VALUES (1, 2, (3,4), ROW(5, 6, 7))) as abc;
 ----
 (1,2,"(3,4)","(5,6,7)")
+
+# MirScalarExpr::reduce() should transform
+# Literal(record(c1, c2)) = record_create(e1, e2)
```

Comment:
nit: same as above - you are comparing a record against a literal list, not a literal record.

Reply:
Fixed